__tests__/integration/fixtures/bug-include-path-edit-uri/mkdocs.yml

```diff
@@ -0,0 +1,11 @@
+site_name: "Example"
+site_description: "Description Here"
+repo_url: https://github.com/backstage/mkdocs-monorepo-plugin
+edit_uri: edit/fix-edit-page-url/__tests__/integration/fixtures/bug-include-path-edit-uri/docs/
+
+plugins:
+  - monorepo
+
+nav:
+  - Home: "index.md"
+  - API: "!include api/mkdocs.yml"
```

Comment:
It might also be nice to have a test fixture that covers the `*include` syntax (recently introduced) as well.